core/preprocessing.py

```diff
@@ -30,10 +34,34 @@ def split_claims_accept_reject(features, label):
     reject_indices = label[label == 0].index
     return (features.iloc[accept_indices], label.iloc[accept_indices]), (features.iloc[reject_indices], label.iloc[reject_indices])
 
+def convert_label_binary(label):
+    return label.mask(label > 0, 1, inplace=False)
+
 def is_categorical_column(column):
     return (column.dtype == 'object'
         or column.nunique() < MIN_REAL_FEATURE_UNIQUE_VALUES)
 
+def encode_feature(feature):
+    return pd.get_dummies(
+        feature,
+        columns=[feature.name],
+    )
+
+def expand_dataset_indexed(dataset):
+    subsets = []
+    for column in dataset.columns:
+        feature = dataset[column]
+        is_encoded = is_categorical_column(feature)
+        if is_encoded:
+            subset = encode_feature(feature)
+            subset_columns = {subset_column: f'{column}_{index}' for index, subset_column in enumerate(subset.columns)}
+            subset.rename(columns=subset_columns, inplace=True)
+            subsets.append(subset)
+        else:
+            subsets.append(feature)
+
+    return pd.concat(subsets, axis=1)
```

Comment:
Had to add this so we don't get the garbage column names after encoding standardized data.